tests/test_taskipy.py

```diff
@@ -26,12 +26,23 @@ def tearDown(self):
         for tmp_dir in self._tmp_dirs:
             tmp_dir.clean()
 
-    def run_task(self, task: str, args: List[str] = None, cwd=os.curdir) -> Tuple[int, str, str]:
+    def run_task(
+        self,
+        task: str,
+        args: Optional[List[str]] = None,
```

Comment:
note: mypy linting changes. The type hint is saying `args` is a list of strings but then is setting it to `None`.